packages/debug/package.json

```diff
@@ -8,7 +8,6 @@
     "@theia/core": "^0.12.0",
     "@theia/editor": "^0.12.0",
     "@theia/filesystem": "^0.12.0",
-    "@theia/json": "^0.12.0",
```

Comment:
it seems it was added as part of https://github.com/eclipse-theia/theia/pull/3663

Reply:
Is it an issue to remove it? I'd understand that if an application wants to support additional features for `json` they'd either include `@theia/json` extension or the builtin `@theia/vscode-builtin-json` plugin explicitly.

In any case, I thought the idea was to slowly phase out language support via our own extensions and instead use the builtins.

Perhaps we should notify of `recommended` extensions when including certain extensions (`preferences`, `debug`) but not explicitly pull them as it offers more flexibility to application developers.